fitbenchmarking/fitbenchmarking
# Fill out workflow docs

#### Description of Work

Fixes #530 


#### Testing Instructions

1. Generate the docs
2. Read the changed section to ensure they are clear


Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
